app/src/main/java/com/toandoan/circleci/MainActivity.java

```diff
@@ -1,13 +1,18 @@
 package com.toandoan.circleci;
 
+import android.content.Context;
```

Comment:
ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports